Dockerfile

```diff
@@ -49,6 +49,7 @@ RUN \
 
 COPY logback-include.xml /go-server/config/logback-include.xml
 COPY install-gocd-plugins /usr/local/sbin/install-gocd-plugins
+COPY git-clone-config /usr/local/sbin/git-clone-config
```

Comment:
Please don't check-in the Dockerfile. As I mentioned before, our CI will take care of updating the Dockerfile if Dockerfile.erb has been changed.

Reply:
done